ExampleMod/Items/Weapons/ExampleFlamethrower.cs

```diff
@@ -0,0 +1,69 @@
+using Microsoft.Xna.Framework;
+using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+using ExampleMod.Projectiles;
+using static Terraria.ModLoader.ModContent;
+
+namespace ExampleMod.Items.Weapons
+{
+	public class ExampleFlamethrower : ModItem
+	{
+		public override string Texture => "Terraria/Item_" + ItemID.Flamethrower;
+
+		public override void SetStaticDefaults()
+		{
+			Tooltip.SetDefault("Shoots a jet of cursed flames\nHas a 66% chance to not consume gel\nThis is a modded flamethrower.");
+		}
+
+		public override void SetDefaults()
+		{
+			item.damage = 69; //The item's damage.
+			item.ranged = true;
+			item.width = 54;
+			item.height = 16;
+			item.useTime = 4;
+			item.useAnimation = 20; //Shoots out 5 jets of fire in one shot
+			item.useStyle = ItemUseStyleID.HoldingOut;
+			item.noMelee = true; //So the item's animation doesn't do damage
+			item.knockBack = 2;
+			item.value = 10000;
+			item.color = new Color(61, 252, 3); //Makes the item color green.
+			item.rare = ItemRarityID.Green; //Sets the item's rarity.
+			item.UseSound = SoundID.Item34;
+			item.autoReuse = true;
+            item.shoot = ProjectileType<FlamethrowerProj>();

```

Comment:
Fix your indentation.

Reply:
Highly certain this is a GitHub artifact, for some reason it ruins indentation.

Reply:
GitHub doesn't cause this kind of issue. Inconsistencies in tabbing do (i.e. mixed spaces and tabs).